iron-ajax.html

```diff
@@ -126,7 +126,7 @@
        * and a `Content-Type` header is specified in the `headers` attribute,
        * the `headers` attribute value will take precedence.
        *
-       * @attribute contenttype
+       * @attribute content-type
```

Comment:
I've been using the property versions (e.g. `contentType`) to doc properties. Is the attribute version the better thing to use? @nevir @garlicnation